src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/LaboratoryInvestigationResultTest.java

```diff
@@ -80,4 +136,53 @@ void testValidate() {
                 .hasMessage("RSL: Attribute measurementUnit is required")
         );
     }
+
+    @Test
+    void testValidateCodedResult() {
```

Comment:
This has 4 different tests in it; can we split them out into different test methods?